### ISSUE TITLE
Accessibility: Color Contrast: Luminosity ratio is less than 3:1 for focus indicator on ToolStrip Buttons with System RenderMode

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#5503]
* .NET Core Version: 
.Net SDK 6.0.100-rc.1.21417.19

* Have you experienced this same bug with .NET Framework?: 
Yes

* More Info:
This issue cannot repro when setting ToolStrip/StatusStrip's RenderMode property to Professional or ManagerRenderMode

**Minimal repro:**
1. Open attached app in VS
[ToolStipButtonSystemRenderer.zip](https://github.com/dotnet/winforms/files/7030031/ToolStipButtonSystemRenderer.zip)
2. Build and run app
3. Use Accessibility Insight to detect the checked toolStripButton color contrast

**Expected behavior:**
The focus color(Light blue) for toolStripButton with 'System' RenderMode is contrast enough, greater than 3:1 

**Actual behavior:**
The focus color(Light blue) for toolStripButton with 'System' RenderMode is not contrast enough:
![image](https://user-images.githubusercontent.com/26474449/130412226-496792aa-6e97-4d0c-aab1-14d43b816c07.png)

**Comment by Jenna-Zheng** [comment]
Verified this issue with latest .NET 7.0.100-alpha.1.21479.21 build, it is fixed that The focus color(Light blue) for toolStripButton with 'System' RenderMode is contrast enough, greater than 3:1.
![image](https://user-images.githubusercontent.com/64949616/135394159-db71def7-2661-49da-b42b-d901146fa931.png)
